trieste/models/gpflow/models.py

```diff
@@ -94,7 +93,7 @@ def __init__(
             kernel when calling :meth:`trajectory_sampler`. We use a default of 1000 as it
             typically perfoms well for a wide range of kernels. Note that very smooth
             kernels (e.g. RBF) can be well-approximated with fewer features.
-        :param used_decoupled_sampler: If True use a decoupled random Fourier feature sampler, else
+        :param use_decoupled_sampler: If True use a decoupled random Fourier feature sampler, else
```

Comment:
its strange that these errors are not caught when building docs, @uri-granta we noticed this before, did you perhaps implement something to catch them?

Reply:
perhaps, but it will be at the bottom of my to do list :)